pipelines/optimus/Optimus.wdl

```diff
@@ -19,44 +21,51 @@ workflow Optimus {
   meta {
     description: "The optimus 3' pipeline processes 10x genomics sequencing data based on the v2 chemistry. It corrects cell barcodes and UMIs, aligns reads, marks duplicates, and returns data as alignments in BAM format and as counts in sparse matrix exchange format."
   }
-  # version of this pipeline
-  String version = "optimus_v1.4.0"
-
-  # Sequencing data inputs
-  Array[File] r1_fastq
-  Array[File] r2_fastq
-  Array[File]? i1_fastq
-  String sample_id
-
-  # organism reference parameters
-  File tar_star_reference
-  File annotations_gtf
-  File ref_genome_fasta
-
-  # 10x parameters
-  File whitelist
-  # tenX_v2, tenX_v3
-  String chemistry = "tenX_v2"
-
-  # environment-specific parameters
-  String fastq_suffix = ""
+  
+  input {
+    # version of this pipeline
+    String version = "optimus_v1.4.0"
+
+    # Sequencing data inputs
+    Array[File] r1_fastq
+    Array[File] r2_fastq
+    Array[File]? i1_fastq
+    String sample_id
+
+    # organism reference parameters
+    File tar_star_reference
+    File annotations_gtf
+    File ref_genome_fasta
+
+    # 10x parameters
+    File whitelist
+    # tenX_v2, tenX_v3
+    String chemistry = "tenX_v2" 
+
+    # environment-specific parameters
+    String fastq_suffix = ""
+    
+    # Emptydrops lower cutoff
+    Int emptydrops_lower = 100
+    
+    # Set to true to override input checks and allow pipeline to proceed with invalid input
+    Boolean force_no_check = false
+
+    # If true produce the optional loom output
+    Boolean output_loom = false
+
+    # Set to true to override input checks and allow pipeline to proceed with invalid input
+    Boolean force_no_check = false
```

Comment:
Looking at the diff it looks like this line was duplicated when `output_loom` was moved. Was this change accidental?

Reply:
Thank you for pointing this out! I've removed this redundant input.